Dockerfile

```diff
@@ -209,6 +209,11 @@ RUN curl -fL -o /tmp/${LIBDNET}.zip https://github.com/dugsong/libdnet/archive/$
 RUN cd $ROOTFS && cd usr/local/lib && ln -s libdnet.1 libdumbnet.so.1 &&\
     cd $ROOTFS && ln -s lib lib64
 
+# TCL 7 doesn't ship with libtirpc.so.1 Dummy it up so the VMware tools work
+# again, taken from:
+# https://github.com/boot2docker/boot2docker/issues/1157#issuecomment-211647607
+RUN cd $ROOTFS/usr/local/lib && ln -s libtirpc.so.3.0.0 libtirpc.so.1 && ldconfig
```

Comment:
Would it also work to just `ln -s libtirpc.so libtirpc.so.1` to insulate ourselves a little bit from future changes?  (`ldconfig` shouldn't be necessary either -- that'd update the LD cache for the build environment, which shouldn't be looking inside `$ROOTFS` anyhow)